### PR TITLE
Read a Market Proposal doc [CT-953]

### PR DESCRIPTION
**Pull request by zachfc**
Adds a page on interpreting a new market proposal JSON document

Intent is mostly for market makers to create programmatic trading software. Users may find it useful to better understand new markets

### CODE REVIEW

File: pages/users-governance/reading_new_market_proposal.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| Reference Price | `reference_price` | Starting price of the proposed asset on the exchange. A user proposing a new market should set this based on the price of the asset at the time they create the proposal. | yes |
+| Liquidity Tier | `liquidity_tier` | Liquidity tier of the proposed asset. A user proposing a new market should set this based on [dYdX liquidity tier guidelines](../users-governance/functionalities#liquidity-tiers). | yes |
+| Atomic Resolution | `atomic_resolution` | Precision of the size of the coin.  | no |
+| Minimum Exchanges | `min_exchanges` | Number of exchanges required to list this asset. | no |
+| Minimum Price Change PPM | `min_price_change_ppm` | The minimum price change that causes the oracle price to update. | no |
+| Exponent | `exponent` | Number of decimal places to use to show prices.  | no |
+| Step Base Quantums | `step_base_quantums` | Minimum amount by which you can increase or decrease an order. Same as `step_size`. | no |
+| Subticks Per Tick | `subticks_per_tick` | TODO Determines the subticks per tick. | no |
+| Quantum Conversion Exponent | `quantum_conversion_exponent` | TODO Determines the subtick size. | no |
```
Comment (zachfc) [inline pages/users-governance/reading_new_market_proposal.md:91]: Please remind me what quantum_conversion_exponent represents/ point me to where I can find this info 🙏

File: pages/users-governance/reading_new_market_proposal.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+| Name | Field | Description | Proposer Input |
+| ---- | ---- | ------------------- | -------- |
+| Reference Price | `reference_price` | Starting price of the proposed asset on the exchange. A user proposing a new market should set this based on the price of the asset at the time they create the proposal. | yes |
+| Liquidity Tier | `liquidity_tier` | Liquidity tier of the proposed asset. A user proposing a new market should set this based on [dYdX liquidity tier guidelines](../users-governance/functionalities#liquidity-tiers). | yes |
+| Atomic Resolution | `atomic_resolution` | Precision of the size of the coin.  | no |
+| Minimum Exchanges | `min_exchanges` | Number of exchanges required to list this asset. | no |
+| Minimum Price Change PPM | `min_price_change_ppm` | The minimum price change that causes the oracle price to update. | no |
+| Exponent | `exponent` | Number of decimal places to use to show prices.  | no |
+| Step Base Quantums | `step_base_quantums` | Minimum amount by which you can increase or decrease an order. Same as `step_size`. | no |
+| Subticks Per Tick | `subticks_per_tick` | TODO Determines the subticks per tick. | no |
```
Comment (zachfc) [inline pages/users-governance/reading_new_market_proposal.md:90]: Please remind me what subticks_per_tick represents/ point me to where I can find this info 🙏

Comment (yujin512) [reply pages/users-governance/reading_new_market_proposal.md:90]: subticks_per_tick determine the "tick size" for each market. tick size in human readable value is subtick in human readable value * subticks_per_tick. To calculate "subtick in human readable value" there's some math involving quantum conversion exponent and other values! discussion on the tick size calculation is here: https://www.notion.so/dydx/Market-Parameters-for-V4-markets-d678dbc594bf483293e829dcb57466f5?pvs=4#8dfedeb4e1944b219ceed4fc8b77ddba

**Review by yujin512 — COMMENTED** [review]
left some comments!

**Review by matthewdowney — APPROVED** [review]

**Pull request merged**